packages/lean-imt/src/lean-imt.ts

```diff
@@ -330,17 +330,31 @@ export default class LeanIMT<N = bigint> {
      * It imports an entire tree by initializing the nodes without calculating
      * any hashes. Note that it is crucial to ensure the integrity of the tree
      * before or after importing it.
-     * The tree must be empty before importing.
+     * @param hash The hash function used to create nodes.
      * @param nodes The stringified JSON of the tree.
+     * @param reviver A function that transforms the results. This function is called for each node of the tree and can
```

Comment:
Maybe point out that this is as defined by JSON.parse, so that's where to look for more details on how this works?
Without that dependency, I would've been asking why the reviver isn't more strongly typed.

Reply:
In the end I made that function much simpler. Can you take a look again?

```ts
// Convert node types to numbers.
LeanIMT.import<number>(hash, nodes, Number) 
```